inline_console.go

```diff
@@ -0,0 +1,396 @@
+package otto
+
+// newConsoleManual is a manually implemented "fork" of newConsole which is generated code.
+func (rt *runtime) newConsoleManual(console Console) *object {
```

Comment:
I added `runtime.newConsoleManual` which is a modified copy of `runtime.newConsole` generated by `tools/gen-jscore`

As of now I don't yet understand how to modify the generator so that it changes the relevant lines

https://github.com/robertkrimen/otto/blob/master/inline.go#L8272-L8275

Help anyone? I think we don't want to manually edit `inline.go`...